src/monaco.ts

```diff
@@ -142,14 +154,69 @@ export async function createModelReference (resource: URI, content?: string): Pr
   return (await StandaloneServices.get(ITextModelService).createModelReference(resource)) as IReference<ITextFileEditorModel>
 }
 
```

Comment:
For my understanding: Can you explain why you did this: "refactor: only use StandaloneKeybindingService for editors"
Would this lead to too many unused keyboard services?

Reply:
One cannot write a review comment against a specific commit message as far as I know, that's why I attached it to some code.

Reply:
There will be only 2 services, the Standalone and the Workbench, just like before.

The only difference is that the standalone one will only be used by Standalone editors (required so the `addAction` api still works) and the Workbench for everything else. The workbench quickaccess code requires to have access to the Workbench keydinging service directly

Reply:
Thank you.